test/functional/mempool_spendcoinbase.py

```diff
@@ -22,12 +22,7 @@ def __init__(self):
         super().__init__()
         self.num_nodes = 1
         self.setup_clean_chain = False
-
-    def setup_network(self):
-        # Just need one node for this test
```

Comment:
meh. Should be clear from the `self.num_nodes` assignment in `__init__()`. I don't think we need this comment (and if we do need it, then there should be a similar comment in every other test case.